flow/e2e/cancel_table_addition_test.go

```diff
@@ -0,0 +1,910 @@
+package e2e
+
+import (
+	"fmt"
+	"time"
+
+	connmongo "github.com/PeerDB-io/peerdb/flow/connectors/mongo"
+	connpostgres "github.com/PeerDB-io/peerdb/flow/connectors/postgres"
+	"github.com/PeerDB-io/peerdb/flow/connectors/utils"
+	"github.com/PeerDB-io/peerdb/flow/generated/protos"
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/v2/bson"
+	"go.mongodb.org/mongo-driver/v2/mongo/options"
+	"google.golang.org/protobuf/proto"
+)
+
+func (s APITestSuite) checkPublicationTables(
+	publicationName string,
+	includedTables []*utils.SchemaTable,
+	cancelledTables []*utils.SchemaTable,
+) {
+
+	pubTables, err := s.pg.PostgresConnector.GetTablesFromPublication(
```

Comment:
If we're getting all tables, can just take `includedTables` as an argument, would also account for strays